package.json

```diff
@@ -21,13 +21,18 @@
     "lint": "eslint . --quiet --ext .json --ext .js",
     "test": "jest",
     "build": "rollup -c",
-    "prepare": "yarn build"
+    "install": "yarn build"
```

Comment:
While prepare is a lifecycle hook, it either doesn't work when run or is not running.

Reply:
do we need to run yarn build at all now if it's only us consuming things?